README.md

```diff
@@ -124,5 +138,20 @@ directory.
 [snyk]: https://snyk.io/test/github/checkstyle/checkstyle:pom.xml?targetFile=pom.xml
 [snyk img]: https://snyk.io/test/github/checkstyle/checkstyle/badge.svg
 
+[flattr]:https://flattr.com/submit/auto?fid=g39d10&amp;url=http%3A%2F%2Fcheckstyle.sourceforge.net
+[flattr img]:https://button.flattr.com/flattr-badge-large.png
+
+[liberapay]:https://liberapay.com/checkstyle/
+[liberapay img]:https://liberapay.com/assets/widgets/donate.svg
+
+[bountysource]:https://www.bountysource.com/teams/checkstyle/issues
+[bountysource img]:https://api.bountysource.com/badge/team?team_id=3568&style=bounties_posted
+
+[salt.bountysource]:https://salt.bountysource.com/teams/checkstyle
+[salt.bountysource img]:https://img.shields.io/bountysource/team/checkstyle/activity.svg?label=salt.bountysource
+
+[backers.opencollective]:https://opencollective.com/checkstyle/
+[backers.opencollective img]:https://opencollective.com/checkstyle/backers/badge.svg
 
-              
+[sponsors.opencollective]:https://opencollective.com/checkstyle/
+[sponsors.opencollective img]:https://opencollective.com/checkstyle/sponsors/badge.svg
```

Comment:
I can't tell for sure from web GUI. Does this file still end with an empty line?

Reply:
confirmed, there is empty line at the end.